tutorials/tutorial_3_build_functions.py

```diff
@@ -187,23 +187,43 @@ def process_results(experiment, platform, output_path):
 
 def plot_results(output_path):
     """
-    The following code uses some tutorial-helper functions to plot the results
+    The following code uses the emodpy_hiv plotting utilities to plot the results
```

Comment:
Nice to use the plotting utilities from emodpy-hiv instead of some helper functions in the tutorial.